lib/skywalking/client.lua

```diff
@@ -72,110 +60,54 @@ function Client:startBackendTimer(backend_http_uri)
     end
 end
 
--- Register service
-function Client:registerService(metadata_buffer, backend_http_uri)
+function Client:reportServiceInstance(metadata_buffer, backend_http_uri)
     local log = ngx.log
     local DEBUG = ngx.DEBUG
     local ERR = ngx.ERR
 
     local serviceName = metadata_buffer:get('serviceName')
+    local serviceInstanceName = metadata_buffer:get('serviceInstanceName')
 
     local cjson = require('cjson')
-    local serviceRegister = require("register").newServiceRegister(serviceName)
-    local serviceRegisterParam = cjson.encode(serviceRegister)
+    local reportInstance = require("management").newReportInstanceProperties(serviceName, serviceInstanceName)
+    local reportInstanceParam = cjson.encode(reportInstance)
 
     local http = require('resty.http')
     local httpc = http.new()
-    local res, err = httpc:request_uri(backend_http_uri .. '/v2/service/register', {
+    local res, err = httpc:request_uri(backend_http_uri .. '/v3/management/reportProperties', {
         method = "POST",
-        body = serviceRegisterParam,
+        body = reportInstanceParam,
         headers = {
             ["Content-Type"] = "application/json",
         },
     })
 
     if not res then
-        log(ERR, "Service register fails, " .. err)
+        log(ERR, "Instance report fails, " .. err)
```

Comment:
better style: `log(ERR, "Instance report fails, ", err)`

it is a simple and better performance.